TennisKata/TennisGameTests.cs

```diff
@@ -0,0 +1,191 @@
+using NUnit.Framework;
+
+namespace TennisKata
+{
+    [TestFixture]
+    public class TennisGameTests
+    {
+        [Test]
+        public void TennisGame_WhenStartingAGame_MatchScoreIsZeroToZero()
+        {
+            // Arrange
+            var tennisMatch = CreateTennisMatch(0, 0, 0, 0, 0, 0);
```

Comment:
Done. Trochę to skomplikowało tworzenie testów, ale wydaje mi się, że teraz jest jasno i czytelnie.